telethon/network/__init__.py

```diff
@@ -7,5 +7,5 @@
 from .mtprotosender import MTProtoSender
 from .connection import (
     ConnectionTcpFull, ConnectionTcpAbridged, ConnectionTcpObfuscated,
-    ConnectionTcpIntermediate, ConnectionHttp
+    ConnectionTcpIntermediate, ConnectionHttp,Connection
```

Comment:
Please add a space after that comma, although I don't know why is this re-export needed.